src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -1507,9 +1471,8 @@ void VideoDecoder::createSwsContext(
 // PTS <-> INDEX CONVERSIONS
 // --------------------------------------------------------------------------
 
-int VideoDecoder::getKeyFrameIndexForPts(
-    const StreamInfo& streamInfo,
-    int64_t pts) const {
+int VideoDecoder::getKeyFrameIndexForPts(int64_t pts) const {
+  const StreamInfo& streamInfo = streamInfos_.at(activeStreamIndex_);
```

Comment:
Using `.at()` instead of `[]` to preserve the `const` qualifier of the method. I would otherwise get:


````
  /home/nicolashug/dev/torchcodec/src/torchcodec/decoders/_core/VideoDecoder.cpp:1475:65: error: passing ‘const std::map<int, facebook::torchcodec::VideoDecoder::StreamInfo>’ as ‘this’ argument discards qualifiers [-fpermissive]
   1475 |   const StreamInfo& streamInfo = streamInfos_[activeStreamIndex_];
        |                                                                 ^
```

@scotts is there a more idiomatic way of dealing with this?